.github/workflows/development-environment.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     timeout-minutes: 20
     strategy:
       matrix:
-        os: [ macos-11.0 ]
+        os: [ macos-latest ]
```

Comment:
At this point, I'm more interested in getting this job running sooner than testing on Big Sur.
I prefer we test this on the most available Mac pool.